### DISCUSSION TITLE
Add 2FA support

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#161] [Feature requests]
For security reasons, it would be great, to have 2FA support. For example TOTP / OTP or FIDO U2F / Webauthn, etc.

**Comment by JulianPrieber** [comment]
https://github.com/search?q=laravel+2fa&type=Repositories

**Reply by MagicLike** [reply]
What about these?:
2FA: https://github.com/stephenafamo/laravel-2fa-starter
Recovery code generator: https://github.com/antonioribeiro/recovery

**Reply by JulianPrieber** [reply]
It's probably smart to choose a dependency that is actively maintained.

I'm a bit confused why Spatie hasn't made something like this
https://spatie.be/open-source?search=&sort=-downloads

**Reply by MagicLike** [reply]
The problem: those were one of the most used ones without Google...
Maybe a deeper search will help...

**Comment by nobaraos12** [comment]
2FA support would be quite nice

**Comment by trymeouteh** [comment]
TOTP 2FA would be nice for additional account security

**Comment by taivlam** [comment]
It would also be nice to offer passkeys as an option for 2FA, such as how [GitHub](https://docs.github.com/en/authentication/authenticating-with-a-passkey/about-passkeys) and [Bitwarden](https://bitwarden.com/help/login-with-passkeys/) have implemented passkeys for their respective users.
